docs/usage/configuration-options.md

```diff
@@ -1979,6 +1979,30 @@ Use the syntax `!/ /` like the following:
 }
 ```
 
+### matchCurrentAge
+
+Use this field if you want to match packages that satisfy the specified age range.
+
+For example, if you want to group packages which are older than 2 months:
+
+```json
+{
+  "packageRules": [
+    {
+      "matchCurrentAge": "> 2 months",
+      "groupName": "vital"
+    }
+  ]
+}
+```
+
+The `matchCurrentAge` string must start with one of `>`, `>=`, `<` or `<=`.
+Only one date part is supported, e.g. you _cannot_ do `> 1 year 1 month` and instead should do `> 13 months`.
+
+<!-- prettier-ignore -->
+!!! note
+    We recommend using these words hour(s), day(s), week(s), month(s) and year(s) to specify time formats.
```

Comment:
Because:

> The `matchCurrentAge` string must start with one of `>`, `>=`, `<` or `<=`.

I think we can rewrite to something like this:

```suggestion
    We recommend you use the words hour(s), day(s), week(s), month(s) and year(s) in your time ranges.
```